Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml

```
@@ -170,4 +182,27 @@
       - PosterLegitSafetyMothHardhat
       - PosterLegitSafetyMothSSD
       - PosterLegitOppenhopper
+      # Cats start
+      - PosterLegitEfficiency
+      - PosterLegitGreatFood
+      - PosterLegitTurnOnSensors 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)